src/components/sections/items/LinkSectionItem.tsx

```diff
@@ -29,73 +29,83 @@ type Props = SectionItemProps<{
-  const linkSectionItemStyle = useStyles();
-  const {theme} = useTheme();
-  const themeColor = theme.interactive[interactiveColor].default;
-  const iconEl =
-    isNavigationIcon(icon) || !icon ? (
-      <NavigationIcon mode={icon} fill={themeColor.text} />
-    ) : (
-      icon
-    );
-  const disabledStyle = disabled ? linkSectionItemStyle.disabled : undefined;
-  const accessibilityWithOverrides = disabled
-    ? {...accessibility, accessibilityHint: undefined}
-    : accessibility;
-  const accessibilityLabel =
-    text + screenReaderPause + (subtitle ? subtitle + screenReaderPause : '');
-  return (
-    <PressableOpacity
-      accessible
-      accessibilityRole="link"
-      onPress={disabled ? undefined : onPress}
-      disabled={disabled}
-      accessibilityLabel={
-        label
-          ? `${accessibilityLabel} ${t(LabelInfoTexts.labels[label])}`
-          : accessibilityLabel
-      }
-      accessibilityState={{disabled}}
-      style={[topContainer, {backgroundColor: themeColor.background}]}
-      testID={testID}
-      {...accessibilityWithOverrides}
-    >
-      <View style={[style.spaceBetween, disabledStyle]}>
-        <ThemeText
-          style={[contentContainer, {color: themeColor.text}]}
-          type={textType}
+
+export const LinkSectionItem = forwardRef<View, Props>(
+  (
+    {
+      text,
+      onPress,
+      subtitle,
+      label,
+      icon,
+      accessibility,
+      disabled,
+      textType,
+      testID,
+      interactiveColor = 'interactive_2',
+      ...props
+    },
+    forwardedRef,
+  ) => {
+    const {t} = useTranslation();
+    const {contentContainer, topContainer} = useSectionItem(props);
+    const style = useSectionStyle();
+    const linkSectionItemStyle = useStyles();
+    const {theme} = useTheme();
+    const themeColor = theme.interactive[interactiveColor].default;
+    const iconEl =
+      isNavigationIcon(icon) || !icon ? (
+        <NavigationIcon mode={icon} fill={themeColor.text} />
+      ) : (
+        icon
+      );
+    const disabledStyle = disabled ? linkSectionItemStyle.disabled : undefined;
+    const accessibilityWithOverrides = disabled
+      ? {...accessibility, accessibilityHint: undefined}
+      : accessibility;
+    const accessibilityLabel =
+      text + screenReaderPause + (subtitle ? subtitle + screenReaderPause : '');
+    return (
+      <PressableOpacity
+        accessible
+        accessibilityRole="link"
+        onPress={disabled ? undefined : onPress}
+        disabled={disabled}
+        accessibilityLabel={
+          label
+            ? `${accessibilityLabel} ${t(LabelInfoTexts.labels[label])}`
+            : accessibilityLabel
+        }
+        accessibilityState={{disabled}}
+        style={[topContainer, {backgroundColor: themeColor.background}]}
+        testID={testID}
+        {...accessibilityWithOverrides}
+      >
+        <View
+          ref={forwardedRef}
+          collapsable={false}
```

Comment:
I have a feeling this ref should be applied to the `PressableOpacity` instead of the `View`. The `PressableOpacity` is the component that is accessible, and most often when we are passing refs it is to focus something for the screen reader.

Reply:
Thanks, I was putting it on the View to test whether the ref works or not, since it works, it should be moved to the `PressableOpacity` 👍